lib/active_remote/attributes.rb

```diff
@@ -1,5 +1,33 @@
 module ActiveRemote
   module Attributes
+    extend ActiveSupport::Concern
+    include ActiveModel::AttributeMethods
```

Comment:
This is a direct holdover from active_attr, and the ability it has to be used piecemeal.

https://github.com/cgriego/active_attr/blob/master/lib/active_attr/attributes.rb#L24

Certainly will be refactoring this in the next pull request, since we do not need the ability to use pieces.